src/views/PersonList.vue

```diff
@@ -1,35 +1,80 @@
 <template>
   <div class="wrapper">
     <div class="content">
-      <h1>People</h1>
-
+      <div class="people-count">{{ items.length }} lovely people</div>
       <p v-for="profile in items" :key="profile.slug">
-        <Avatar :profile="profile" />
+        <PeopleAvatar :profile="profile" />
       </p>
     </div>
-    <div class="sidebar"></div>
+    <div class="sidebar">
+      <TextInput
+        v-model="searchQuery"
+        icon="search"
+        placeholder="search"
+        @input="handleSearchInput"
+      />
+    </div>
   </div>
 </template>
 
 <script>
 import api from '../api'
-import Avatar from '@/components/Avatar'
+import TextInput from '../components/forms/TextInput.vue'
+import PeopleAvatar from '@/components/PeopleAvatar'
+import { popQuery, debounce } from '@/utils'
 
 export default {
   name: 'PersonList',
   metaInfo: {
     title: 'People',
   },
-  components: { Avatar },
+  components: { PeopleAvatar, TextInput },
   data() {
     return {
       items: [],
+      searchQuery: '',
     }
   },
+  computed: {},
   async created() {
     const { results: items } = await api.getProfiles()
     this.items = items
   },
-  methods: {},
+  methods: {
+    refetch() {
+      this.items = []
+      this.fetchItems(0)
+    },
+    async fetchItems(offset) {
+      let query = {
+        offset,
+        search: this.searchQuery,
+      }
+      query = query = Object.entries(query).reduce((a, [k, v]) => (v ? ((a[k] = v), a) : a), {})
+      const { results, count } = await api.getProfiles(query)
+      this.items = [...this.items, ...results]
+      this.offset = this.offset + results.length
+      this.count = count
+    },
+    handleSearchInput: debounce(function (query) {
+      if (query) {
+        this.$router.replace({ query: { ...this.$route.query, search: query } })
+      } else {
+        popQuery(this.$router, this.$route.query, 'search')
+      }
+      this.refetch()
+    }, 200),
+  },
 }
 </script>
+
+<style lang="scss" scoped>
+.people-count {
```

Comment:
Use `small muted sans utility class instead?

https://github.com/aecworks/aec.works-web/blob/960d9fde19f3ae482a550ff4d5917bc80803a3d6/src/views/CompanyList.vue#L24

For media query, could we not use the sass mixins which are used around the project?
```
@include for-large-up {
...
}
```

Reply:
Oh right, that makes sense. I'll get rid of that scoped style then. 

For the media query, I'm not sure if there's a way to import sass to js...is there?

Reply:
Oh right right. I'll change that up. I actually have never used mixins, so that didn't even cross my mind haha. I'll put those in and re-commit.